frontend/components/ActivityDetails/InstallDetails/SoftwareInstallDetailsModal/SoftwareInstallDetailsModal.tests.tsx

```diff
@@ -0,0 +1,163 @@
+import React from "react";
+import { render, screen } from "@testing-library/react";
+import { renderWithSetup } from "test/test-utils";
+import { createMockSoftwareInstallResult } from "__mocks__/softwareMock";
+import { StatusMessage, ModalButtons } from "./SoftwareInstallDetailsModal";
+
+describe("SoftwareInstallDetailsModal - StatusMessage component", () => {
```

Comment:
You can actually nest `describes` for a nice visualization when you run the tests, so could do
something like
`describe("SoftwareInstallDetailsModal", () => {
  describe("StatusMessage"), ...);
  describe("ModalButtons", ...);
});`
something

Reply:
ooo! good to know @jacobshandling